e2e/tests/api/rdf/admin!custom-rdf-edit.spec.ts

```diff
@@ -0,0 +1,36 @@
+const { chromium } = require('playwright');
```

Comment:
ah, that is why you made the e2e changes.

I would not want to use playwright for API tests. Instead use Java see for examples WebApiSmokeTests

Reply:
Noted. Yeah Playwright had some issues as in VS code it worked but local cmd/CI failed. Will remove code here though create separate PR in case someone in the future needs logged-in e2e tests (with example test).